components/lib/crash/README.md

```diff
@@ -82,13 +82,33 @@ Add a `MozillaSocorroService` instance to your `CrashReporter` in order to uploa
 ```Kotlin
 CrashReporter(
     services = listOf(
-		MozillaSocorroService(applicationContext, "your app name"),
+		MozillaSocorroService(applicationContext, "your app name")
     )
 ).install(applicationContext)
 ```
 
 ⚠️ Note: Currently only native code crashes get uploaded to Socorro. Socorro has limited support for "uncaught exception" crashes too, but it is recommended to use a more elaborate solution like Sentry for that.
 
+### Sending crash reports to Glean
```

Comment:
Is this a good location to put a reminder that applications using the GleanCrashReporterService are required to undergo Data Collection Review for the crash counts they will be collecting?

Reply:
I think so, this should be the launch point for someone wishing to use the `GleanCrashReporterService` in their app, and they would hopefully see a note about requesting review for their particular app/population.

Reply:
Added an additional note:
⚠️ Note: Applications using the `GleanCrashReporterService` are required to undergo Data Collection Review for the crash counts that they will be collecting.